server/graphql/v2/interface/AccountWithHost.ts

```diff
@@ -0,0 +1,63 @@
+import { GraphQLBoolean, GraphQLFloat, GraphQLInterfaceType, GraphQLNonNull } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+import { isNil } from 'lodash';
+
+import { HOST_FEE_STRUCTURE } from '../../../constants/host-fee-structure';
+import { hostResolver } from '../../common/collective';
+import { HostFeeStructure } from '../enum/HostFeeStructure';
+import { Host } from '../object/Host';
+
+export const AccountWithHostFields = {
+  host: {
+    description: 'Returns the Fiscal Host',
+    type: Host,
+    resolve: hostResolver,
+  },
+  hostFeesStructure: {
+    description: 'Describe how the host charges the collective',
+    type: HostFeeStructure,
+    resolve: async (account, _, req): Promise<HOST_FEE_STRUCTURE | null> => {
+      if (!account.HostCollectiveId) {
+        return null;
+      } else if (isNil(account.hostFeePercent)) {
+        return HOST_FEE_STRUCTURE.DEFAULT;
+      } else {
+        const host = await req.loaders.Collective.byId.load(account.HostCollectiveId);
+        return account.hostFeePercent === host.hostFeePercent
+          ? HOST_FEE_STRUCTURE.DEFAULT
+          : HOST_FEE_STRUCTURE.CUSTOM_FEE;
+      }
+    },
+  },
+  hostFeePercent: {
```

Comment:
That might be used to expose the `hostFeePercent` of the host itself.

Reply:
`Host` has its own implementation of this field, because it doesn't inherit from `AccountWithHost` there's no risk of overriding.